File: Assets/VirtualCity/ProtoDefine/Java/system/UserInfoMap.java
package com.kingston.jforgame.server.game.system.entity;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

public class UserInfoMap {
    @Protobuf(order = 1)
    private int infoType;
    @Protobuf(order = 2)
    private String info;

    public int getInfoType() {
        return infoType;
    }

    public void setInfoType(int infoType) {
        this.infoType = infoType;
    }

    public String getInfo() {
        return info;
    }

    public void setInfo(String info) {
        this.info = info;
    }

    public UserInfoMap() {
    }

    public UserInfoMap(short infoType, String info) {
        this.infoType = infoType;
        this.info = info;
    }
}

File: Assets/VirtualCity/ProtoDefine/UserInfoMap.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class UserInfoMap {
[ProtoMember(1)]
    public int infoType;
[ProtoMember(2)]
    public string info;

    public int getInfoType() {
        return infoType;
    }

    public void setInfoType(int infoType) {
        this.infoType = infoType;
    }

    public string getInfo() {
        return info;
    }

    public void setInfo(string info) {
        this.info = info;
    }

    public UserInfoMap() {
    }

    public UserInfoMap(short infoType, string info) {
        this.infoType = infoType;
        this.info = info;
    }
}
}